src/main/java/com/snowflake/kafka/connector/DefaultConnectorConfigValidator.java

```diff
@@ -21,9 +21,13 @@ public class DefaultConnectorConfigValidator implements ConnectorConfigValidator
       new KCLogger(DefaultConnectorConfigValidator.class.getName());
 
   private final StreamingConfigValidator streamingConfigValidator;
+  private final StreamingConfigValidator icebergConfigValidator;
```

Comment:
nit: some idea
We could have a list of validators and later we could just use streams to call operations on them like
`invalidConfigParams = validators.map(StreamingConfigValidator::validate).toList`

Reply:
that would also mean couple of lines less in the future ;p

Reply:
I think it would make more sense to do it after we refactor Snowpipe specific validations to have a similar interface. But in general yes.